src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link, useLocation } from 'react-router-dom';
+
+import {
+  usePeopleDispatch,
+  usePeopleState,
+} from '../components/store/PeopleContext';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+
+export const PeoplePage: React.FC = () => {
+  const location = useLocation();
+
+  const { people, error, loading } = usePeopleState();
+  const dispatch = usePeopleDispatch();
+
+  React.useEffect(() => {
+    const currentPath = location.pathname;
+    const basePeoplePath = '/people';
+
+    if (currentPath.startsWith(basePeoplePath)) {
+      dispatch({
+        type: 'SET_ERROR',
+        payload: '',
+      });
+      dispatch({
+        type: 'SET_LOADING',
+        payload: true,
+      });
+      getPeople()
+        // eslint-disable-next-line
+        .then(people => {
+          dispatch({
+            type: 'SET_PEOPLE',
+            payload: people,
+          });
+          dispatch({
+            type: 'SET_LOADING',
+            payload: false,
+          });
+        })
+        .catch(() => {
+          dispatch({
+            type: 'SET_ERROR',
+            payload: 'Something went wrong',
+          });
+        });
+    }
+  }, []);
+
+  const getSlugByName = (name: string) => {
+    const human = people.find(person => person.name === name);
+
+    return human?.slug || null;
+  };
+
+  const setNameColor = (sex: string) => {
+    if (null) {
+      return;
+    }
+
+    return cn({
+      'has-text-danger': sex === 'f',
+      'has-text-success': sex === 'm',
+    });
+  };
+
+  if (error) {
+    return (
+      <p data-cy="peopleLoadingError" className="block box has-text-danger">
+        {error}
+      </p>
+    );
+  } else if (!people.length && !loading) {
+    return <p data-cy="noPeopleMessage">There are no people on the server</p>;
+  }
```

Comment:
Let's a little bit improve readability 
```suggestion
  if (error) {
    return (
      <p data-cy="peopleLoadingError" className="block box has-text-danger">
        {error}
      </p>
    );
  } 
  
  if (!people.length && !loading) {
    return <p data-cy="noPeopleMessage">There are no people on the server</p>;
  }
```